app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/bugreport/VpnAppTrackerListInfoCollector.kt

```diff
@@ -49,8 +52,11 @@ class VpnAppTrackerListInfoCollector @Inject constructor(
     }
 
     private fun isUnprotectedByDefault(appPackageId: String): Boolean {
-        return appCategoryDetector.getAppCategory(appPackageId) is AppCategory.Game ||
-            appTrackerRepository.getAppExclusionList().any { it.packageId == appPackageId }
+        return isGame(appPackageId) || appTrackerRepository.getAppExclusionList().any { it.packageId == appPackageId }
+    }
+
+    private fun isGame(packageName: String): Boolean {
+        return appCategoryDetector.getAppCategory(packageName) is AppCategory.Game && !appTpFeatureConfig.isEnabled(AppTpSetting.ProtectGames)
```

Comment:
nit: can we have create an `appTpFeatureConfig.isDisabled` to make it clearer?

Reply:
Will do in different PR so refactor all usages if `isEnabled`